.github/workflows/eslint.yml

```
@@ -97,7 +97,7 @@ jobs:
           fi
 
       - name: Upload analysis results to GitHub
-        uses: github/codeql-action/upload-sarif@38697555549f1db7851b81482ff19f1fa5c4fedc # v4.34.1
+        uses: github/codeql-action/upload-sarif@c10b8064de6f491fea524254123dbe5e09572f13 # v4.35.1
```

YAMLlint (1.38.0)

[warning] too few spaces before comment: expected 2

(comments)